packages/react/passport/src/PassportContext.tsx

```diff
@@ -0,0 +1,187 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import React, {
+  createContext, useCallback, useContext, useEffect, useMemo,
+  useState,
+} from 'react';
+import { setPassportClientId, track } from '@imtbl/metrics';
+import {
+  Passport, PassportModuleConfiguration,
+  UserProfile,
+} from '@imtbl/passport';
+
+type ZkEvmReactContext = {
+  passportInstance: Passport;
+  isLoggedIn: boolean;
+  isLoading: boolean;
+  withWallet: boolean;
```

Comment:
What's the use case for `withWallet`? If the consumer specifies whether they want the wallet initialised or not, then the consumer would already know if the wallet has been initialised